website/layouts/partials/_default/sidemenu.html

```diff
@@ -5,34 +5,64 @@
 </button>
 
 <nav class="docs-menu collapse d-lg-block">
-    <h5>Thanos</h5>
-    <div class="list-group list-group-flush">
-        {{ range .Site.Menus.thanos }}
-            <a class="list-group-item list-group-item-action list-group-item-thanos py-1" href="{{ .URL }}">{{ .Title }}</a>
-        {{ end }}
+    <div class="sidemenu-thanos">
+        <input type="checkbox" id="menuinput-thanos" class="menuinput"{{if eq .Section ""}} checked{{end}} />
+        <label for="menuinput-thanos" class="menulabel-thanos">
+            <h5>Thanos</h5>
+            <img class="caret-expand" src="/expand.svg" alt="expand" />
+        </label>
+        <div class="list-group list-group-flush">
+            {{ range .Site.Menus.thanos }}
+                <a class="list-group-item list-group-item-action list-group-item-thanos py-1" href="{{ .URL }}">{{ .Title }}</a>
+            {{ end }}
+        </div>
     </div>
-    <h5 class="mt-3">Components</h5>
-    <div class="list-group list-group-flush">
-        {{ range .Site.Menus.components }}
-            <a class="list-group-item list-group-item-action list-group-item-thanos py-1" href="{{ .URL }}">{{ .Title }}</a>
-        {{ end }}
+    <div class="sidemenu-thanos">
+        <input type="checkbox" id="menuinput-components" class="menuinput"{{if eq .Section "components"}} checked{{end}} />
+        <label for="menuinput-components" class="menulabel-thanos">
+            <h5 class="mt-3">Components</h5>
+            <img class="caret-expand mt-3" src="/expand.svg" alt="expand" />
+        </label>
+        <div class="list-group list-group-flush">
+            {{ range .Site.Menus.components }}
+                <a class="list-group-item list-group-item-action list-group-item-thanos py-1" href="{{ .URL }}">{{ .Title }}</a>
+            {{ end }}
+        </div>
     </div>
-    <h5 class="mt-3">Operating</h5>
-    <div class="list-group list-group-flush">
-        {{ range .Site.Menus.operating }}
-            <a class="list-group-item list-group-item-action list-group-item-thanos py-1" href="{{ .URL }}">{{ .Title }}</a>
-        {{ end }}
+    <div class="sidemenu-thanos">
+        <input type="checkbox" id="menuinput-operating" class="menuinput"{{if eq .Section "operating"}} checked{{end}} />
```

Comment:
Nice touch on these

```
{{if eq .Section "operating"}} checked{{end}}
```

:+1:

Reply:
Thanks. I am new to Go (and go templating) and I got the idea after poking around some hugo themes. Hope I got this right haha.